app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -61,6 +60,19 @@ https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/pe
   iface = @host.provision_interface
   appstream_present = false
 -%>
+# This kickstart file was rendered from the Foreman provisioning template "<%= @template_name %>".
+# for <%= @host %> running <%= @host.operatingsystem.name %> <%= os_major %> <%= @arch %>
+<% if organization %># in <%= organization %> <% end %>
+<% if location %># at <%= location %> <% end %>
+<% begin -%>
+<%
+  # Katello variables
+  lifecycle_environment = @host.info['parameters']['lifecycle_environment']
+  content_view = @host.info['parameters']['content_view']
+  content_source = @host.content_source.hostname
```

Comment:
Please use `fetch` too.

For the last line, method `content_source` will only be available if Katello plugin is installed, this template must work without Katello too. Method `try` is not safe-mode compatible, but you can try this:

```suggestion
  content_source = @host&.content_source&.hostname
```

Finally, enclose the whole block with `if plugin_present?('katello')` because this template must work without Katello too.